spec/index.html

```diff
@@ -116,7 +116,7 @@ <h2>Introduction</h2>
     typically define a class <code>eg:Book</code> with an attribute called
     <code>eg:author</code> of type <code>eg:Person</code>. Using the RDF
     approach, it is easy for others to subsequently define additional
-    properties with a domain of eg:<code>Document</code> or a range of
+    properties with a domain of <code>eg:Document</code> or a range of
```

Comment:
Where is `prefix eg:` declared? Just as `e.g.` and `i.e.` should be avoided in text, I strongly suggest that we change the current `eg:` prefix to `example:` or at worst `ex:` and include the `prefix {ex|example}:` declaration in whatever docs contain uses of it.